### PR TITLE
Docs: Introduce lincense and add credit to strv for inspiration

### PR DESCRIPTION
**Pull request by literat**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```
Comment (adamkudrna) [inline LICENSE:1]: Why not a more common MIT? As far as I know they are [compatible](https://opensource.stackexchange.com/a/582) and MIT is more adopted in the npm universe.

Comment (literat) [reply LICENSE:1]: Great inspiration for this project was [STRV's code-quality-tools repo](https://github.com/strvcom/code-quality-tools). Their code is under this license which demands same license of forks or derived projects. I do not want to pretend that this project is not based on others projects or other developers work.

I have no problem with MIT but on the other hand I would like to give a credit to my resources.

Comment (adamkudrna) [reply LICENSE:1]: Got it, so it's a price for inspiration 🙂. OK then!

**Review by adamkudrna — APPROVED** [review]

**Review by adamkudrna — CHANGES_REQUESTED** [review]
We almost forgot: the license needs to be updated in all `package.json` files.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Copyright (c) 2019, LMC s.r.o.
```
Comment (adamkudrna) [inline LICENSE:3]: The year is intentionally 2019…?

Comment (literat) [reply LICENSE:3]: Copy&paste bug.

**Comment by literat** [comment]
> We almost forgot: the license needs to be updated in all `package.json` files.

Thanks, I've just updated those files.

**Review by adamkudrna — APPROVED** [review]

**Pull request merged**